crates/bevy_ui/src/render/mod.rs

```diff
@@ -342,11 +348,24 @@ pub fn extract_uinode_images(
             continue;
         }
 
+        let ui_logical_viewport_size = camera_query
+        .get(camera_entity)
+        .ok()
+        .and_then(bevy_render::camera::Camera::logical_viewport_size)
+        .unwrap_or(Vec2::ZERO)
+        // The logical window resolution returned by `Window` only takes into account the window scale factor and not `UiScale`,
+        // so we have to divide by `UiScale` to get the size of the UI viewport.
+        / ui_scale.0;
```

Comment:
Is there a reason this was moved?

Reply:
It was obtained by the camera_entity in the main world, so it should be executed before mapping